src/Navigation/Xamarin.Forms/Navigation.Xamarin.Forms/NavigationServiceExtensions.cs

```diff
@@ -121,18 +113,14 @@ public static class NavigationServiceExtensions
         /// <param name="preCreateInstance">Indicates wether an instance of the view should be created and cached before it is displayed</param>
         /// <returns><see cref="CodeMonkeys.Navigation.Xamarin.Forms.Models.NavigationRegistration"/></returns>
         public static NavigationRegistration Register<TViewModelInterface, TPhoneView, TTabletView, TDesktopView>(
-            this INavigationService navigationService,
-            bool preCreateInstance = false)
+            this INavigationService navigationService)
 
             where TViewModelInterface : class, IViewModel
             where TPhoneView : Page
             where TTabletView : Page
             where TDesktopView : Page
         {
-            var registrationInfo = new NavigationRegistration<TViewModelInterface, TPhoneView, TTabletView, TDesktopView>
-            {
-                PreCreateInstance = preCreateInstance
-            };
+            var registrationInfo = new NavigationRegistration<TViewModelInterface, TPhoneView, TTabletView, TDesktopView>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 124 characters long line (which is greater than 120 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5598919)